pkg/workload/schemachange/query_util.go

```diff
@@ -18,9 +18,38 @@ import (
 	"github.com/jackc/pgx/v5"
 )
 
-var (
+const (
+	// descJSONQuery returns the JSONified version of all descriptors in the
+	// current database joined with system.namespace.
+	//
+	// id::int | schema_id::int | name::text | descriptor::json
+	descJSONQuery = `SELECT
+		descriptor.id,
+		"parentSchemaID" AS schema_id,
+		namespace.name AS name,
+		crdb_internal.pb_to_json('desc', descriptor) AS descriptor
+	FROM system.descriptor
+	JOIN system.namespace ON namespace.id = descriptor.id
+	WHERE "parentID" = (SELECT id FROM system.namespace WHERE name = current_database() AND "parentID" = 0)
+	`
+
+	// enumDescsQuery returns the JSONified version of all enum descriptors in
+	// the current database.
+	//
+	// [descJSONQuery] must be bound to the name "descriptors".
+	//
+	// id::int | schema_id::int | name::text | descriptor::json
+	enumDescsQuery = `SELECT id, schema_id, name, descriptor->'type' AS descriptor FROM descriptors WHERE descriptor ? 'type'`
```

Comment:
what does "WHERE descriptor ? 'type'" mean here? Tried some online searching but didn't find a definite answer (is it a placeholder? If so, what value is substituted later for this placeholder?)